bikeshare_app/templates/checkin.html

```diff
@@ -99,15 +98,15 @@ <h2 class="ui horizontal divider header">
 
 {% block additionalScripts %}
 <script>
-    $('div[name="damageTextArea"]').hide();
+    $('div[name="damageTextArea"]').css('display','none');
```

Comment:
This doesn't fix the issue of it happening on the page after it loads the content. Set it in the stylesheet appropriately.